types/global.d.ts

```diff
@@ -75,6 +75,7 @@ interface KeyboardEvent {
 declare namespace NodeJS {
   interface Global {
     __REDUX_DEVTOOLS_EXTENSION_COMPOSE__: any;
+    ga: UniversalAnalytics.ga | any | undefined;
```

Comment:
curious why the `any | undefined`? can make `ga` optional by adding a `?`:

```jsx
ga?: UniversalAnalytics.ga;
```

Reply:
will try that, I needed to be able to set `global.ga = undefined` in my tests